ontology/EBUCorePlus/ebucoreplus.owl

```diff
@@ -13707,6 +13696,26 @@ ec:Rating rdf:type owl:Class ;
                      "Classement"@fr ,
                      "Rating"@en .
 
+###  http://www.ebu.ch/metadata/ontologies/ebucoreplus#Ratio
+ec:Ratio rdf:type owl:Class ;
+          rdfs:subClassOf [ rdf:type owl:Restriction ;
+                            owl:onProperty ec:numerator ;
+                            owl:minQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                            owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                            owl:onDataRange xsd:integer
+                          ] ,
+                          [ rdf:type owl:Restriction ;
+                            owl:onProperty ec:denominator ;
+                            owl:minQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                            owl:maxQualifiedCardinality "1"^^xsd:nonNegativeInteger ;
+                            owl:onDataRange xsd:integer ;
+                            xsd:minInclusive 1
```

Comment:
Question about the restriction :        xsd:minInclusive 1
Don't you think you should encapsulate this restriction within a datatype definition ? To be  OWL 2 compliant ?

Reply:
Sorry for the long review time! During our last Editorial Committee meeting we have concluded to pack this into a new issue and use a new class `Ratio` for two of the three attributes mentioned above. I have now re-analysed the comments and would also use it for the Attribute `frameRate`.
But as I am starting to do this, I am stumbling across this question of aro-max. What do you think, @MarcAntoine-Arnaud ? Should you encapsulate the restriction within a datatype definitrion as proposed by @aro-max ? If we can resolve this question quickly, I might be able to review, merge and commit your pull request quickly as well.